src/index.css

```
@@ -1667,6 +1675,1092 @@ html.palette-dark .send-asset-menu {
+  min-height: var(--size-touch-lg);
+  border-radius: var(--radius-xl);
+  background: var(--neutral-50);
+  box-shadow: inset 0 0 0 1px color-mix(in srgb, var(--fg) 4%, transparent);
+}
+
+.swap-search-field span {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)